CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
 
-## [0.17.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.16.1...0.17.0) (2021-02-02)
+## [0.17.1](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.16.0...0.17.1) (2021-02-10)
```

Comment:
Why do you add it here? It should be a new paragarph

Reply:
Should we list 0.17.0 when it has been skipped and never been released?

Reply:
let's merge it to 0.17.1.

Reply:
sorry, my bad, we've released 0.17.0. We have to split the changelog. Thanks